eng/build.py

```diff
@@ -0,0 +1,178 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.

```

Comment:
In the next story about build automation, I will add an acceptance testing criteria suggesting that this script be documented in the main readme, or in a developer readme, so that developers know how to use it for localhost command line operations.

Reply:
It would be nice if we at least mention the python version that we're using when running this script, or even better to specify the minimum python version.

Reply:
Ok I saw the validation for Python 3.8, but we could add it to the .teamcity readme

Reply:
Done.